app/views/child_benefit_tax/_results.html.erb

```diff
@@ -63,12 +63,16 @@
 
         <p>You need to pay a <a href="/child-benefit-tax-charge">tax charge on the Child Benefit</a> you’ve claimed.</p>
         <p>To pay the tax charge you must fill in a Self Assessment tax return each tax year. Follow these steps:</p>
-        <ol>
-          <li><a href="/register-for-self-assessment">Register for Self Assessment</a> if you don’t already fill in a tax return - you should do this by <%= sa_register_deadline(@calculator) %>.</li>
-          <li>Fill in the tax return and include the amount of Child Benefit received for the tax year.</li>
-          <li>Send the tax return.</li>
-          <li>Pay the tax you owe.</li>
-        </ol>
+        <%= render "govuk_publishing_components/components/list", {
+          list_type: "number",
+          visible_counters: true,
+          items: [
+              sanitize("<a href=\"/register-for-self-assessment\">Register for Self Assessment</a> if you don’t already fill in a tax return - you should do this by #{sa_register_deadline(@calculator)}."),
```

Comment:
Aha. This makes sense to sanitise since it's a link but yeah, the other items here I don't reckon need sanitising.